opennlp-tools/src/main/java/opennlp/tools/sentdetect/SentenceSampleStream.java

```diff
@@ -33,8 +33,17 @@
  */
 public class SentenceSampleStream extends FilterObjectStream<String, SentenceSample> {
 
-  public SentenceSampleStream(ObjectStream<String> sentences) {
+  private final Character defaultEOS;
+  private final char[] eos;
+
+  public SentenceSampleStream(ObjectStream<String> sentences, char[] eos, Character defaultEOS) {
```

Comment:
Not sure this is a good place to add in a default eos char. Can't this be done somewhere else?